package.json

```diff
@@ -105,7 +105,11 @@
     "webpack-version-file-plugin": "^0.4.0",
     "whatwg-fetch": "^3.0.0",
     "wicked-good-xpath": "^1.3.0",
-    "xml-js": "^1.6.11"
+    "xml-js": "^1.6.11",
+    "@grpc/proto-loader": "^0.3.0",
```

Comment:
Do we really need these deps in here? This feels wrong

Reply:
Without the GRPC deps we cannot have the client working.

Reply:
But the GRPC deps are being sucked in via the sdk dependency - at least this is what I'm expecting.
Works that way with any module. Let me know if there is anything to help out with.

Reply:
Ah sorry.. This was a previous attempt just for testing and I forgot to remove it. Again, this is just to kick start the discussion and is definitely not for merging. I have things half baked in many places.

Reply:
No problem at all - I'm just having a close eye :eyes: ;-)